packages/flutter/test/cupertino/nav_bar_test.dart

```diff
@@ -928,8 +928,8 @@ void main() {
     await tester.pumpWidget(
       WidgetsApp(
         color: const Color(0xFFFFFFFF),
-        onGenerateRoute: (RouteSettings settings) {
-          return CupertinoPageRoute<void>(
+        onGenerateRoute: <T>(RouteSettings settings) {
```

Comment:
This is a breaking change and will need to follow our policy on making breaking changes.